lms/djangoapps/courseware/module_render.py

```diff
@@ -713,9 +713,9 @@ def rebind_noauth_module_to_user(module, real_user):
         wrappers=block_wrappers,
         get_real_user=user_by_anonymous_id,
         services={
-            'i18n': ModuleI18nService(),
             'fs': FSService(),
             'field-data': field_data,
+            'i18n': ModuleI18nService,
```

Comment:
I think, we can remove it from here as `LmsModuleSystem` already initializing this service in `__init__`